### DISCUSSION TITLE
Mobile app

### DISCUSSION THREAD

**Discussion by RedstoneWizard08** [modrinth/.github#221] [Feature requests]
The idea: a mobile modrinth app for quick project management and browsing.

To me, on a phone browser, the website is very slow (I know that it's much faster on a PC and is amazing, but it's slow on mobile). A mobile app would be much faster and I find that I would love to manage my projects, view analytics, and browse mods on my phone, and a mobile app would make that much easier.

An additional idea: some sort of integration with Pojav Launcher (possibly)?

**Comment by ignissak** [comment]
The web app seems very snappy for me because it is SSR enabled. And I don't know if Modrinth team has a capability to develop native Android/iOS app at the moment. A middle-way solution like PWA shouldn't be a problem though but the performance might not improve that much.

**Comment by Grayvox** [comment]
Tauri's mobile app support is currently a WIP, but that could possibly be a way to approach this in the future. 
There's also the option to use Ionic Capacitor for this since it'd be just the website in a mobile form. I would have to say this is likely not a priority for them, but it would be cool to see in the future!
